_literate/basics/syntax.jl

```diff
@@ -1,8 +1,8 @@
 #=
-- Julia's arrays default to column vector behaviour
-- Julia uses brackets to access elements and indices start at 1
-- Slices are inclusive of endpoints, and `end` means end.
-- Steps can be passed in the middle
+- Julia's arrays default to column vector behavior
```

Comment:
```suggestion
- Julia's arrays default to column vector behaviour
```
https://internet.psych.wisc.edu/wp-content/uploads/532-Master/532-UnitPages/Unit-13/Grammarist_Behavior_NoDate.pdf